utils.py

```diff
@@ -1055,11 +1162,15 @@ def partition(iterable, predicate=bool, enumerated=False):
         themselves.
     """
     if enumerated:
-        iterable = enumerate(iterable)
+        new_iterable = enumerate(iterable) # type: Any
```

Comment:
Why `Any`?

Reply:
This variable here is of type enumerate and 'enum' is not a type in MyPy. So used this. What do you think?

Reply:
Isn't it like `Iterator[(int, T)]` where `T` is the type of `iterable`?

Reply:
Did it. Thanks!